modules/ob-release-notes-1-0.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies:
+// * builds/ob-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="ob-release-notes-1-0_{context}"]
+= Release notes for {builds-title} General Availability 1.0
+
+{builds-title} General Availability (GA) 1.0 is now available on {ocp-product-title} 4.14.
+
+[id="new-features-1-0_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-title} 1.0.
+
+[id="builds-new-features-1-0_{context}"]
+=== Builds
+
+* This release supports the `buildah` and `source-to-image` build strategies.
++
+[NOTE]
+====
+The `buildpacks` build strategy is currently in Developer Preview. It is planned to make this strategy generally available for use in a future release.
```

Comment:
Link to https://access.redhat.com/support/offerings/devpreview?

Swap "it" for "Red Hat"?